coverage_analysis/coverage_tests/spiral_print_test.py

```diff
@@ -0,0 +1,28 @@
+from typing import Iterable
+from numpy import random
+from random import randint
+import sys
+sys.path.append('../')
+from data_structure.coverage_tool import Coverage_tool
+from data_structure.coverage_data_structure import CoverageData
+from functions_analysed.spiral_print_coverage import spiralPrintTest
+
+def test_wrong_matrix_1(coverage: CoverageData):
+    spiralPrintTest([], coverage)
+    f = open('spiralPrint.txt', 'r')
+    assert f.read() == "Not a valid matrix\n"
+    f.close()
+
+def test_wrong_matrix_2(coverage: CoverageData):
+    # find a test that enter branches 3, 4 and 5
+    pass
```

Comment:
This test is empty, but I guess we can always add code later if we find a suitable test case.